src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -506,6 +510,36 @@ public void delete() throws IOException {
     }
 
     /**
+     * Sort orders for listing forks
+     */
+    public static enum Sort { NEWEST, OLDEST, STARGAZERS }
+
+    /**
+     * Lists all the direct forks of this repository, sorted by {@link Sort#NEWEST Sort.NEWEST}
+     */
+    public PagedIterable<GHRepository> listForks() {
+      return listForks(null);
```

Comment:
Because I don't know the default. It is defined by github, and they do not expose it, except in their documentation.
So my javadoc is wrong, it should read "sorted by whatever github api defined as the default sort order, currently NEWEST"